src/status_im/ui2/screens/chat/actions.cljs

```diff
@@ -311,23 +333,34 @@
                          (notification-actions item inside-chat?)
                          (destructive-actions item)]])
 
-(defn contact-actions [{:keys [public-key added] :as contact}]
-  [drawer/action-drawer [[(view-profile-entry public-key)
-                          (when added
+(defn contact-actions [{:keys [public-key] :as contact} {:keys [chat-id admin?] :as extra-data}]
+  (let [current-pk (rf/sub [:multiaccount/public-key])]
```

Comment:
Could you rename this binding to `current-public-key` or `current-pub-key`? There are many usages of `pk` referring to *private key* in the literature, so this is always very confusing to me.